css-three-column-layout/styles.css

```diff
@@ -0,0 +1,56 @@
+.container {
+  background-color: #1996da;
+  margin-left: auto;
+  margin-right: auto;
+  padding-left: 10%;
+  padding-right: 10%;
+  padding-top: 2%;
+}
+
+.row {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.column {
+  flex-basis: 25%;
+
+  /* border: 1px, solid, black; */
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
```

Comment:
Few things here. Like I mentioned above, I would expect this `.column` class to just have display flex and the flex-direction of column. Other things should be in separate classes.

If you have 3 columns, should it be 25%? and for 3 columns, what would we name a class that make the flex basis for 3?

You can also make `align-items` into it's own utility class such as:
```
items-center {
  align-items: center;
}
```
and now you can reuse that class anywhere you need to align the flex items along the cross axis.